diff --git a/node_modules/react-native/src/private/webapis/dom/events/Event.js b/node_modules/react-native/src/private/webapis/dom/events/Event.js
index f97ead5..2e7dcdb 100644
--- a/node_modules/react-native/src/private/webapis/dom/events/Event.js
+++ b/node_modules/react-native/src/private/webapis/dom/events/Event.js
@@ -49,11 +49,6 @@ export default class Event {
   static +AT_TARGET: 2;
   static +BUBBLING_PHASE: 3;
 
-  +NONE: 0;
-  +CAPTURING_PHASE: 1;
-  +AT_TARGET: 2;
-  +BUBBLING_PHASE: 3;
-
   _bubbles: boolean;
   _cancelable: boolean;
   _composed: boolean;
diff --git a/node_modules/react-native/src/private/webapis/errors/DOMException.js b/node_modules/react-native/src/private/webapis/errors/DOMException.js
index b759632..a5dc9bd 100644
--- a/node_modules/react-native/src/private/webapis/errors/DOMException.js
+++ b/node_modules/react-native/src/private/webapis/errors/DOMException.js
@@ -99,32 +99,6 @@ export default class DOMException extends Error {
   static +INVALID_NODE_TYPE_ERR: 24;
   static +DATA_CLONE_ERR: 25;
 
-  +INDEX_SIZE_ERR: 1;
-  +DOMSTRING_SIZE_ERR: 2;
-  +HIERARCHY_REQUEST_ERR: 3;
-  +WRONG_DOCUMENT_ERR: 4;
-  +INVALID_CHARACTER_ERR: 5;
-  +NO_DATA_ALLOWED_ERR: 6;
-  +NO_MODIFICATION_ALLOWED_ERR: 7;
-  +NOT_FOUND_ERR: 8;
-  +NOT_SUPPORTED_ERR: 9;
-  +INUSE_ATTRIBUTE_ERR: 10;
-  +INVALID_STATE_ERR: 11;
-  +SYNTAX_ERR: 12;
-  +INVALID_MODIFICATION_ERR: 13;
-  +NAMESPACE_ERR: 14;
-  +INVALID_ACCESS_ERR: 15;
-  +VALIDATION_ERR: 16;
-  +TYPE_MISMATCH_ERR: 17;
-  +SECURITY_ERR: 18;
-  +NETWORK_ERR: 19;
-  +ABORT_ERR: 20;
-  +URL_MISMATCH_ERR: 21;
-  +QUOTA_EXCEEDED_ERR: 22;
-  +TIMEOUT_ERR: 23;
-  +INVALID_NODE_TYPE_ERR: 24;
-  +DATA_CLONE_ERR: 25;
-
   #name: string;
   #code: number;
 
